tests/unit/classifier/LDA_unittest.cc

```diff
@@ -200,10 +200,10 @@ TEST(LDA, num_classes_in_labels_exception)
 	SGVector<float64_t> lab{1, -1, 2};
 	SGMatrix<float64_t> feat(1, 3);
 	auto labels = some<CMulticlassLabels>();
-	labels->set_labels(lab);
+	labels->put("labels", lab);
 	auto features = some<CDenseFeatures<float64_t>>(feat);
 	auto lda = some<CLDA>(0, SVD_LDA);
-	lda->set_labels(labels);
+	lda->put("labels", dynamic_cast<CLabels*>(labels.get()));
```

Comment:
is there the need to use .get? I think there is a put for some, or?

Reply:
Yes, there is a `put` for `Some` indeed.